packages/notebook-extension/schema/tracker.json

```diff
@@ -270,7 +276,8 @@
         "readOnly": false,
         "insertSpaces": true,
         "tabSize": 4,
-        "wordWrapColumn": 80
+        "wordWrapColumn": 80,
+        "rulers": [88]
       }
     },
     "markdownCellConfig": {
```

Comment:
I often like to keep my markdown to a max-line length as well. I see no reason not to allow `"rulers"` in markdown and raw cells. Furthermore, it would be nice to keep this schema in line with the `editorConfig` interface.

Reply:
Also, the lack of "rulers" in the markdown/raw config, coupled with it being on by default in the JS `editorConfig` seems to make it so that they are always on in markdown and raw cells, with no way to turn them off.

Reply:
Thanks for the review.
And thanks for highlighting the others configuration.